src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
```

Comment:
Why use union and enum together? Please pick one for consistency.

Reply:
If I use an enum for color, I get this in the tests.
![image](https://github.com/mate-academy/ts_figures_classes/assets/26518146/716a572c-2694-4e48-9ce5-7185827e6921)